com.unity.multiplayer.mlapi/Editor/ProfilerModuleOverride/ProfilerInfo.cs

```diff
@@ -0,0 +1,126 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Text;
+using MLAPI.Profiling;
+using Unity.Collections.LowLevel.Unsafe;
+using Unity.Profiling;
+using UnityEditorInternal;
+using UnityEngine.Profiling;
+
+namespace ProfilerModuleOverride
+{
+    public static class ProfilerInfo
```

Comment:
Thinking about it, I think we need to move `ProfilerInfo` and the `OnPerformanceTickData` from the other file into the runtime assemblies, or else users will not able to attach their profiler in the editor to a build player. To do this you can use `[RuntimeInitializeOnLoad]` (which is applied to methods and not classes) instead of `[InitializeOnLoad]`

Reply:
This has been fixed and moved elsewhere.